docs/_pages/installation/roborock.md

```diff
@@ -80,8 +80,19 @@ Example: https://dustbuilder.xvm.mit.edu/pkg/s5/v11_002008.fullos.fd043420-6ddb-
 ### Image Building
 It is recommended to use [https://github.com/zvldz/vacuum](https://github.com/zvldz/vacuum) to build the image.
 
-`--valetudo-path` expects a path to a folder containing a binary named `valetudo`.
-Refer to [https://github.com/Hypfer/Valetudo/releases](https://github.com/Hypfer/Valetudo/releases) to fetch the latest valetudo binary.
+`--valetudo-path` expects a path to a folder containing two things:
+
+ * The source code of [Valetudo](https://github.com/Hypfer/Valetudo)
+ * And a binary named `valetudo`.  Refer to [https://github.com/Hypfer/Valetudo/releases](https://github.com/Hypfer/Valetudo/releases) to fetch the latest valetudo binary.
```

Comment:
No need to put the link twice or in brackets if text and target are the same.

Reply:
Fixed.